patches/0063-Allow-DHCP-broadcast-packets.patch

```diff
@@ -0,0 +1,71 @@
+From 760f47667cd0fd2aa4b15ec8864f8e25841e3d00 Mon Sep 17 00:00:00 2001
+From: Nagendrababu <nagendrababu.bonkuri@connectus.ai>
+Date: Fri, 10 Sep 2021 12:47:12 -0400
+Subject: [PATCH] WIFI-3905-AP does not forward dhcp broadcast offer
+
+This patch will add fix to allow DHCP broadcast packets, so that AP can forward DHCP broadcast offer to clients
+---
+ .../A00-Allow-DHCP-braodcast-packet.patch     | 51 +++++++++++++++++++
+ 1 file changed, 51 insertions(+)
+ create mode 100644 target/linux/ipq40xx/patches-4.14/A00-Allow-DHCP-braodcast-packet.patch
+
+diff --git a/target/linux/ipq40xx/patches-4.14/A00-Allow-DHCP-braodcast-packet.patch b/target/linux/ipq40xx/patches-4.14/A00-Allow-DHCP-braodcast-packet.patch
+new file mode 100644
+index 0000000000..ed9ad2a362
+--- /dev/null
++++ b/target/linux/ipq40xx/patches-4.14/A00-Allow-DHCP-braodcast-packet.patch
+@@ -0,0 +1,51 @@
++Index: linux-4.14.187/net/bridge/br_forward.c
++===================================================================
++--- linux-4.14.187.orig/net/bridge/br_forward.c
+++++ linux-4.14.187/net/bridge/br_forward.c
++@@ -20,6 +20,37 @@
++ #include <linux/if_vlan.h>
++ #include <linux/netfilter_bridge.h>
++ #include "br_private.h"
+++#include <linux/ip.h>
+++#include <linux/udp.h>
+++
+++static struct bootp_pkt {	/* BOOTP packet format */
```

Comment:
is there a way not to add this whole structure here and detect the dhcp packets?

Reply:
To detect the DHCP packet there is local structure in the source code. And didn't find any global structure definitions to detect the DHCP packet.